src/app/events/event-active-registration/event-active-registration.component.html

```
@@ -76,12 +76,37 @@
           >
             {{ cancellation.buttonLabel }}
           </button>
         </div>
       }
+      @if (transferActionCopy(registration); as transferAction) {
+        <div class="flex flex-col items-start gap-2 lg:col-span-2">
+          <p class="body-small text-on-surface-variant">
+            {{ transferAction.helperText }}
+          </p>
+          <button
+            mat-stroked-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)